### PR TITLE
Feature/update links

### LINKED ISSUE #6740: Mobile FAQ link is broken
The mobile app has a link to the FAQ from Profile > Need Help? > Frequently Asked Questions.

The FAQ is no longer shown.

It links to https://wiki.status.im/Frequently_Asked_Questions which now redirects to the docs site tech documentation page.

Solution:

Change the link from `https://wiki.status.im/Frequently_Asked_Questions ` to `https://status.im/docs/FAQs.html` in the app.

Needs to be fixed for the 0.9.31 release.

### PR DESCRIPTION
**Pull request by andytudhope**
Fixes #6740

### Summary:

The link to the FAQs in the app is broken. I did a find + replace for all the links I could get, but still haven't been able to find the guilty culprit. Maybe someone better at Clojure could help a brother out. 

#### Platforms (optional)
- Android
- iOS
- macOS
- Linux
- Windows

#### Areas that maybe impacted (optional)
**Functional**
- account recovery
- new account
- user profile updates
- networks
- mailservers
- fleet
- bootnodes

### Steps to test:
- Open Status
- Check that the link under Profile > Need Help? > Frequently Asked Questions points to the right place, which is here: https://status.im/docs/FAQs.html

status: WIP

<blockquote><div><strong><a href="https://status.im/docs/FAQs.html">Status, a Mobile Client for Ethereum 2.0</a></strong></div></blockquote>

### CODE REVIEW

**Review by chadyj — APPROVED** [review]
Thanks for the quick update!

**Comment by lukaszfryc** [comment]
@andytudhope does it all work or you still need some clojure help?

**Comment by andytudhope** [comment]
Long story short: I had a nightmare trying to sync my fork to latest develop, got into merge conflict hell, and ended up deleting it all and starting fresh, so I am not running this locally right now and don't have 30mins to build it all - was hoping someone else could test if they're already set up... Sorry!

**Comment by lukaszfryc** [comment]
Ok, so let's wait for jenkins to check it. After that we will merge

**Review by lukaszfryc — APPROVED** [review]

File: src/status_im/ui/screens/help_center/views.cljs [hunk 1]
```diff
@@ -18,7 +18,7 @@
      [profile.components/settings-item-separator]
      [profile.components/settings-item {:label-kw            :t/faq
                                         :accessibility-label :faq-button
-                                        :action-fn           #(.openURL react/linking "https://wiki.status.im/Questions_around_beta#firstHeading")}]
+                                        :action-fn           #(.openURL react/linking "https://status.im/FAQs.html")}]
```
Comment (lukaszfryc) [inline src/status_im/ui/screens/help_center/views.cljs]: @andytudhope should be `https://status.im/docs/FAQs.html`

**Review by lukaszfryc — CHANGES_REQUESTED** [review]

**Review by flexsurfer — APPROVED** [review]

**Review by lukaszfryc — APPROVED** [review]

**Comment by lukaszfryc** [comment]
@andytudhope works fine. Please merge :)

**Comment by cammellos** [comment]
@andytudhope could you squash the commits please?

**Pull request merged**